b2share/modules/records/utils.py

```diff
@@ -45,3 +43,14 @@ def is_deposit(record):
         bool: True if the record is a deposit record, else False.
     """
     return record.json['$schema'].endswith('#/draft_json_schema')
+
+
+
+
+def list_db_published_records():
+    """A generator for all the published records"""
+    query = RecordMetadata.query.filter(RecordMetadata.json != None)
```

Comment:
How important is this? Currently this fn is not on a critical code path: https://github.com/EUDAT-B2SHARE/b2share/issues/1374